eng/Versions.props

```diff
@@ -7,6 +7,7 @@
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>2</PreReleaseVersionIteration>
     <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
+    <UseVSTestRunner>true</UseVSTestRunner>
```

Comment:
What's the use of this Property ? Secondly, this property should not be present in Version.props

Reply:
This is to use VSTestRunner rather than xunit runner. Reference other dotnet repos, It is present in version.props only.
https://grep.app/search?f.path.pattern=version&q=usevstestrunner